content/en/docs/cluster-administration/cluster-status-monitoring.md

```diff
@@ -0,0 +1,166 @@
+
+![ETCD Monitoring](/images/docs/cluster-administration/cluster-status-monitoring/etcd-monitoring.png)
+
+## APIServer Monitoring
+
+[API Server](https://kubernetes.io/docs/concepts/overview/kubernetes-api/) As the hub for the interaction of all components in the Kubernetes cluster, the following table lists the main indicators monitored by the APIServer.
+
+|Monitoring indicators|Description|
+|---|---|
+|Request Latency|Classified statistics by HTTP request method, the latency of resource request response in milliseconds|
+|Request Per Second|The number of requests accepted by kube-apiserver per second|
+
+![APIServer Monitoring](/images/docs/cluster-administration/cluster-status-monitoring/apiserver-monitoring.png)
+
+## Scheduler Monitoring
+
+[Scheduler](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/) Scheduler is the Kubernetes API of monitor the newly created Pod, and determine which nodes the newly created Pod should run on. It makes this decision based on available data, including the availability of collected resources and the resource requirements of the Pod. Monitoring data for scheduling delays ensure that you can see any delays faced by the scheduler.
+
+|Monitoring indicators|Description|
+|---|---|
+|Attempt Frequency|Including the number of scheduling successes, errors, and failures|
+|Attempt Rate|Including scheduling rate of success, error, and failure|
+|Scheduler latency|End-to-end scheduling delay, which is the sum of scheduling algorithm delay and binding delay|
+
+![Scheduler Monitoring](/images/docs/cluster-administration/cluster-status-monitoring/scheduler-monitoring.png)
+
+## Node Usage Ranking
+
+The node usage ranking is very useful for host monitoring. It supports perform ranking with indicators such as **CPU, Load Average, Memory, Local Storage, inode Utilization, and Pod Utilization**, support ascending and descending order. Administrators can quickly find potential problems or locate a node's insufficient resources by sorting by indicators. 
```

Comment:
It supports perform ranking with indicators such as **CPU, Load Average, Memory, Local Storage, inode Utilization, and Pod Utilization**, support ascending and descending order. Administrators can quickly find potential problems or locate a node's insufficient resources by sorting by indicators.
->
You can sort nodes in ascending and descending order by indicators such as **CPU, Load Average, Memory, Local Storage, inode Utilization, and Pod Utilization**. This enables administrators to quickly find potential problems or identify a node's insufficient resources.